transforms/dec128.js

```diff
@@ -0,0 +1,78 @@
+const DECIMAL_LITERAL = "DecimalLiteral";
+
+const opToName = {
+  '+': 'add',
+  '*': 'mul',
+  '-': 'sub',
+  '/': 'div',
+};
+
+const includedOps = Object.keys(opToName);
+
+const isDecimalOrBinaryExpBare = (t) => (node) => {
+  return isDecimal(node) || t.isBinaryExpression(node);
+};
+
+const areMixedTypesBare = (isDecimalOrBinaryExp) => (a, b) => {
+
+  return (isDecimalOrBinaryExp(a) && !isDecimalOrBinaryExp(b))
+    || !(isDecimalOrBinaryExp(a) && isDecimalOrBinaryExp(b))
+};
+
+const isDecimal = (node) => {
```

Comment:
Another nit: I would call this param `path` rather than `node`. In Babel there is the distinction between "nodes" and "node path". Nodes are "plain object", while paths are the rich wrapper with all the utility methods (such as `path.get("callee")`) and the scope info.